src/components/ColorPalette/ColorPalette.js

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react';
+import { ContentSwitcher, Switch, Dropdown } from 'carbon-components-react';
+import cx from 'classnames';
+import {
+  categoricalLight,
+  oneColorLight,
+  twoColorLight,
+  threeColorLight,
+  fourColorLight,
+  fiveColorLight,
+  categoricalDark,
+  oneColorDark,
+  twoColorDark,
+  threeColorDark,
+  fourColorDark,
+  fiveColorDark,
+  monoColors,
+  divergingColors,
+  alert,
+} from '../../data/data-visualization/palettes';
+import ColorPaletteColor from './ColorPaletteColor';
+import PalettesContainer from './PalettesContainer';
+
+const ColorPalette = ({ type, isMono, isDiverging }) => {
+  // STATES
+  const [continuous, setContinuous] = useState(false);
+  const [dark, setDark] = useState(false);
+  const [groupNumber, setGroupNumber] = useState(1); // used for selected dropdown item
+  let colors;
+
+  // DETERMINE LIGHT/DARK
+  const categorical = dark ? categoricalDark : categoricalLight;
+  const oneColor = dark ? oneColorDark : oneColorLight;
+  const twoColor = dark ? twoColorDark : twoColorLight;
+  const threeColor = dark ? threeColorDark : threeColorLight;
+  const fourColor = dark ? fourColorDark : fourColorLight;
+  const fiveColor = dark ? fiveColorDark : fiveColorLight;
+
+  // SET RENDERED COLORS
+  const [colorGroup, setColorGroup] = useState(oneColor); // used to render type === "grouped" colors
+  if (type === 'sequential' && isMono) {
+    colors = monoColors;
+  } else if (type === 'sequential' && isDiverging) {
+    colors = divergingColors;
+  } else if (type === 'categorical') {
+    colors = categorical;
+  } else if (type === 'alert') {
+    colors = alert;
+  }
+
+  // DROPDOWN STUFF
+  const dropdownItems = [
+    {
+      id: 1,
+      label: '1-Color group',
+    },
+    {
+      id: 2,
+      label: '2-Color group',
+    },
+    {
+      id: 3,
+      label: '3-Color group',
+    },
+    {
+      id: 4,
+      label: '4-Color group',
+    },
+    {
+      id: 5,
+      label: '5-Color group',
+    },
+  ];
+
+  const onDropdownChange = (e) => {
+    // update selected option
+    setGroupNumber(e.selectedItem.id);
+
+    // update colors rendered
+    switch (e.selectedItem.id) {
+      case 1:
+        setColorGroup(oneColor);
+        break;
+      case 2:
+        setColorGroup(twoColor);
+        break;
+      case 3:
+        setColorGroup(threeColor);
+        break;
+      case 4:
+        setColorGroup(fourColor);
+        break;
+      case 5:
+        setColorGroup(fiveColor);
+        break;
+      default:
+    }
+  };
+
+  // SWITCHER STUFF
+  const setTrue = () => {
```

Comment:
Same as `setFalse` maybe we could combine these functions and pass in a variable to determine the behavior (rather than splitting in two)